doc/install_macos.txt

```diff
@@ -0,0 +1,197 @@
+.. include:: css.txt
+
+.. _install_macos:
+
+
+Mac OS Installation Instructions
+################################
+
+There are various ways to install Theano dependencies on a Mac. Here
+we describe the process in detail with Canopy, Anaconda, Homebrew or
+MacPorts but if you did it differently and it worked, please let us
+know the details on the `theano-users`_ mailing-list, so that we can
+add alternate instructions here.
+
+.. _theano-users: http://groups.google.com/group/theano-users?pli=1
+
+In academia: Enthought Canopy
```

Comment:
We always went back and forth considering that conda option. I am not sure what's the "official" opinion about that.